tests/test_input.py

```diff
@@ -18,9 +20,8 @@ def test_input_invalid_working_dir_():
     # Assert path to DICOMs does not exist
     assert not invalid_test_input.working_directory.is_dir()
 
-    invalid_filesystem_input = FilesystemInput(working_directory="NOT_VALID_PATH")
-    # Assert path to DICOMs does not exist
-    assert not invalid_filesystem_input.working_directory.is_dir()
+    with pytest.raises(Exception):
```

Comment:
Any preference for using .raises() over assert() in this particular case? I've always used try/excepts for error handling so be curious to get your insight on raise/raises() :)

Reply:
Yeah I wanted to test here that the function (or class constructor in this case) raises an exception if the directory doesn't exist. This is the pytest syntax to confirm that an exception is raised on the code contained within the with context.

Reply:
Cool I'll keep this in mind then for future tests